src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -55,6 +55,10 @@ exitSpecList.forEach((val) => {
   exitSpecMap[val.code] = val;
 });
 
+// schedule ports range is [20000, 40000)
+const portStart = 20000;
+const portEnd = 40000;
```

Comment:
could you make these two configuration? i.e., allows an admin/operator to specify the range during deployment.

Reply:
Fixed

Reply:
Pls limit the Ephemeral Port Range used by k8s (and other daemons), and rand outside the limit.

Reply:
It's not rest-server part. Synced with yifan. For k8s or other daemons, we need to run command on the host to limit the port range. May need to change it in deployment part